biogrid/libs/grid-simulator/src/lib/validate-inputs/validate-inputs.ts

```diff
@@ -3,7 +3,7 @@ export interface Validatable {
   value: number;
   min?: number;
   max?: number;
-  isInt(): boolean;
+  isPositive: boolean;
```

Comment:
Is this something you could calculate from `value` without having a separate input?

Reply:
I think I could write an assert, however. I would have to check it every time I want to check if the number is positive. I am using it like 4 times.